tools/update_assisted_installer_yaml.py

```diff
@@ -11,22 +12,56 @@
 args = parser.parse_args()
 
 
+def in_all_lists(el: str, lists: list[str]) -> bool:
+    for list in lists:
+        if el not in list:
+            return False
+    return True
+
+
+def find_first_common_element(lists: list[str]) -> str:
+    if len(lists) < 1:
+        raise IndexError("No lists provided. Cannot find common element")
```

Comment:
Seems to be more relevant in this case:
```suggestion
        raise ValueError("No lists provided. Cannot find common element")
```